receiver/mysqlreceiver/scraper.go

```diff
@@ -186,6 +185,17 @@ func (m *mySQLScraper) scrapeGlobalStats(now pcommon.Timestamp, errs *scrapererr
 		case "Connection_errors_tcpwrap":
 			addPartialIfError(errs, m.mb.RecordMysqlConnectionErrorsDataPoint(now, v,
 				metadata.AttributeConnectionErrorTcpwrap))
+		case "Aborted_clients":
+			addPartialIfError(errs, m.mb.RecordMysqlConnectionErrorsDataPoint(now, v,
+				metadata.AttributeConnectionErrorAbortedClients))
+		case "Aborted_connects":
+			addPartialIfError(errs, m.mb.RecordMysqlConnectionErrorsDataPoint(now, v,
+				metadata.AttributeConnectionErrorAborted))
+		case "Locked_connects":
+			addPartialIfError(errs, m.mb.RecordMysqlLockedConnectsDataPoint(now, v))
+			addPartialIfError(errs, m.mb.RecordMysqlConnectionErrorsDataPoint(now, v,
+				metadata.AttributeConnectionErrorLocked))
+		
```

Comment:
@djaglowski I have doubts about this piece of code. Having separate metric for `locked_connects` is not a good solution imho.

MySQL documentation has separate page for [communication errors](https://dev.mysql.com/doc/refman/8.0/en/communication-errors.html) and [locked account](https://dev.mysql.com/doc/refman/8.0/en/account-locking.html), but for me connection to locked account should be measured by `connection.errors` as well.

I'm thinking about deprecating `mysql.locked_connects` and removing in future PRs. WDYT?

Reply:
👍 I agree with your assessment and implementation.